filenames.gypi

```diff
@@ -279,6 +279,8 @@
       'atom/browser/ui/accelerator_util_views.cc',
       'atom/browser/ui/atom_menu_model.cc',
       'atom/browser/ui/atom_menu_model.h',
+      'atom/browser/ui/certificate_trust_mac.mm',
+      'atom/browser/ui/certificate_trust.h',
```

Comment:
Super minor: I think the order here should be swapped, looks like `.` sorts before `_` in other places in this list like `file_dialog.h`.